src/content/blog/astro-ecosystem-fund.mdx

```diff
@@ -0,0 +1,69 @@
+
+Funding comes from our own open-source sponsorship funds, collected and managed through corporate sponsorship and individual donations. We are thankful to our major open-source sponsors for making this possible: [Vercel](https://vercel.com/), [Storyblok](https://www.storyblok.com/), [Sentry](https://sentry.io/), [Netlify](https://www.netlify.com/), [Google Chrome](https://developer.chrome.com/), [and more](https://github.com/withastro/astro#sponsors). Without their support, this initiative would never have been possible.
+
+**Interested in supporting our work? [Sponsor us!](https://www.notion.so/Partner-with-Astro-b05cfd5dcf4248dcbea4a2d81554efed?pvs=21)** 100% of funds raised are invested back into the community. Learn more about our approach to open-source funding on [GitHub](https://github.com/withastro/.github/blob/main/FUNDING.md) and browse our fully transparent project finances on [Open Collective](https://opencollective.com/astrodotbuild).
+
+## From $0 to $200,000
+
+Astro operates as an open-source project. We follow an open governance model, an open roadmap process, and a completely open budgeting process through [Open Collective](https://opencollective.com/astrodotbuild). Our funding is made up entirely of sponsorships and donations to the project, which come from companies and some amazing individuals.
+
+Two years ago we had $0 in our Open Collective. Today, we manage an estimated annual sponsorship budget of $178,374.52 USD with over $200,000 in total funds raised.
+
+Reaching this level of sponsorship didn't happen overnight, and it didn't happen accidentally. Our funding is the result of consistent effort and outreach to large sponsors over the last two years. It's one of the less visible projects within Astro that I'm personally the most proud of.
+
+Early on, a group of us also made the decision to form [The Astro Technology Company](https://astro.build/blog/the-astro-technology-company/). The company provides additional full-time resources to the project, which is another reason why we are able to reserve 100% of our sponsorship for our community of maintainers, contributors, and ecosystem builders.
+
+We don't take this unique privilege lightly. Initiatives like the Astro Ecosystem Fund let us leverage this fortunate position to also give back and support other, less-visible open-source projects in our community.
+
+## Support open-source software
+
+
+<img
+	src="https://imgs.xkcd.com/comics/dependency.png"
+	alt={'xkcd comic about open-source dependencies depicting a complex and barely stable structure made out of various sized blocks labeled as "All modern digital infrastructure". A small block seemingly integral to the building's infrastructure is labeled as "A project some random person in Nebraska has been thanklessly maintaining since 2003."'}
```

Comment:
```suggestion
	alt={'xkcd comic about open-source dependencies depicting a complex and barely stable structure made out of various sized blocks labeled as "All modern digital infrastructure". A small block seemingly integral to the building\'s infrastructure is labeled as "A project some random person in Nebraska has been thanklessly maintaining since 2003."'}
```